imports/email/notifications.tests.js

```diff
@@ -12,6 +12,7 @@ import { Communities } from '/imports/api/communities/communities.js';
 import { Topics } from '/imports/api/topics/topics.js';
 import { Comments } from '/imports/api/comments/comments.js';
 import { Partners } from '/imports/api/partners/partners.js';
+import { ParcelBillings } from '/imports/api/transactions/parcel-billings/parcel-billings.js';
```

Comment:
Nem lett volna eleg a 275 soron a supplier-t atirni memberre hogy fusson?
Csak mert ugye minel tobb dependenciat hozunk be actesztbe annal nehezebb lesz az eletunk. 
Kell ide a ParcelBilling eloiras kirovas logika ebbe a tesztbe?

Reply:
A bills.makeJournalEntries-ben, amit a post hív, van egy olyan sor, hogy         
if (this.relation === 'member') contraAccount += ParcelBillings.findOne(line.billing.id).digit;
és mivel a post methodot teszteljük, hogy hívja az email küldést, ezt nem tudjuk megkerülni.